buildstockbatch/workflow_generator/residential/residential_hpxml_arg_mapping.py

```diff
@@ -63,5 +63,11 @@
         "ReportSimulationOutput": {
             "output_variables": "user_output_variables",
         },
+        "ReportUtilityBills": {
+            "include_annual_bills": "include_annual_bills",
+            "register_annual_bills": "register_annual_bills",
+            "include_monthly_bills": "include_monthly_bills",
+            "register_monthly_bills": "register_monthly_bills",
```

Comment:
This is supposed to be a mapping from yaml to the measure. 
```suggestion
            "include_annual_bills": "register_annual_bills",
            "include_monthly_bills": "register_monthly_bills",
```